src/sail_lean_backend/pretty_print_lean.ml

```diff
@@ -160,6 +160,7 @@ let rec doc_typ ctx (Typ_aux (t, _) as typ) =
       if provably_nneg ctx low then string "Nat" else string "Int"
   | Typ_var kid -> doc_kid ctx kid
   | Typ_app (id, _) -> doc_id_ctor id
+  | Typ_exist (_, _, typ) -> doc_typ ctx typ
```

Comment:
What are the first two components?

Reply:
Then I think it's safe to ignore the constraints and leave the `k_id`s implicit for now